message MessageTaskUpdate
{
    required int32  status = 1;
	repeated string output = 2; 
}
